deploy/kubernetes/helm-charts/altinn-repositories/templates/ingress.yaml

```diff
@@ -0,0 +1,26 @@
+{{- $serviceName := include "altinn-repositories.fullname" . }}
+{{- $containerPort := .Values.image.containerPort }}
```

Comment:
A deployment/pod might map to several ports, this limits us to only using one port per deployment/pod.